lib/pure/asyncftpclient.nim

```diff
@@ -79,7 +79,17 @@
 
 
 import asyncdispatch, asyncnet, nativesockets, strutils, parseutils, os, times
-from net import BufferSize
+from net import BufferSize, SslContext
+
+when defined(ssl):
+  from net import SslHandshakeType, newContext, SslCVerifyMode
+  var defaultSslContext {.threadvar.}: SslContext
+
+  proc getSSLContext(): SslContext =
+    if defaultSSLContext == nil:
+      defaultSSLContext = newContext(verifyMode = CVerifyPeer)
+    result = defaultSSLContext
```

Comment:
This is the same as httpclient afaik, would be nice to clean this up in the future so we don't have each module having its own `defaultSSLContext`. But that can happen in a follow up PR.